memory/provider.go

```diff
@@ -103,17 +103,14 @@ func (mp *Provider) Count() int {
 
 // NeedGC need gc
 func (mp *Provider) NeedGC() bool {
-	if mp.expiration == 0 {
-		return false
-	}
-
 	return true
 }
 
 // GC session garbage collection
 func (mp *Provider) GC() {
 	for _, kv := range mp.memoryDB.D {
-		if time.Now().Unix() >= (kv.Value.(*Store).lastActiveTime + mp.expiration) {
+		store := kv.Value.(*Store)
```

Comment:
Check if expiration is `0` to non remove that session.

Reply:
done